app/server/static/assets/css/admin.css

```diff
@@ -102,3 +102,15 @@
 }
 
 [v-cloak] { display: none; }
+
+
+/* Labels UI */
+
+.random-color-button {
```

Comment:
Given that this class is only used in the labels component, you can include the style directly in the Vue template:

```vue
<template>
...
</template>

<style scoped>
.random-color-button {
  ...
}
</style>

<script>
  ...
</script>
```

The webpack Vue integration takes care of extracting the `<style>` tag from the `.vue` single-file component and injecting it into the page. Additionally, adding the `scoped` attribute to the style block will lead Vue to only apply the rules in that style block to this particular component which means that there will never be issues of affecting other components with the CSS. [More information.](https://vue-loader.vuejs.org/guide/scoped-css.html)